tests/cupy_tests/test_cusparse.py

```diff
@@ -107,6 +108,42 @@ def test_csrmm2_with_c(self):
         testing.assert_array_almost_equal(y, expect)
 
 
+@testing.parameterize(*testing.product({
+    'dtype': [numpy.float32, numpy.float64, numpy.complex64, numpy.complex128],
+    'shape': [(3, 4), (4, 3)]
+}))
+@testing.with_requires('scipy>=1.2.0')
+class TestCsrgeam(unittest.TestCase):
```

Comment:
Sorry, can we add also tests for invalid configurations?

Reply:
No problem!  I will add some unit tests for invalid cases.